core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/ApiServiceSpec.scala

```diff
@@ -191,21 +192,27 @@ trait ApiServiceSpec
 
     val config = SubmissionMonitorConfig(5 seconds, 30 days, true, 20000, true)
     val testConf = ConfigFactory.load()
-    val submissionSupervisor = system.actorOf(
-      SubmissionSupervisor
-        .props(
-          executionServiceCluster,
-          new UncoordinatedDataSourceAccess(slickDataSource),
-          samDAO,
-          gcsDAO,
-          mockNotificationDAO,
-          gcsDAO.getBucketServiceAccountCredential,
-          config,
-          testConf.getDuration("entities.queryTimeout").toScala,
-          workbenchMetricBaseName
-        )
-        .withDispatcher("submission-monitor-dispatcher")
-    )
+    // for tests that don't need it, don't even start any cromwell submission monitors.
+    // this cuts down on resources and on spurious log messages.
+    val submissionSupervisor = if (startSubmissionMonitor) {
+      system.actorOf(
+        SubmissionSupervisor
+          .props(
+            executionServiceCluster,
+            new UncoordinatedDataSourceAccess(slickDataSource),
+            samDAO,
+            gcsDAO,
+            mockNotificationDAO,
+            gcsDAO.getBucketServiceAccountCredential,
+            config,
+            testConf.getDuration("entities.queryTimeout").toScala,
+            workbenchMetricBaseName
+          )
+          .withDispatcher("submission-monitor-dispatcher")
+      )
+    } else {
+      system.actorOf(TestActors.blackholeProps)
+    }
```

Comment:
This change isn't strictly necessary for this PR. As I was debugging the `AvroUpsertMonitorSpec` tests, I kept seeing log message from the submission monitor which got in the way of what I was trying to look at. These were purely extraneous - so I added a way to turn it off for any tests that rely on `ApiServiceSpec`.